services/profile/profile-api/src/main/java/com/softgroup/profile/api/model/Contact.java

```diff
@@ -0,0 +1,36 @@
+package com.softgroup.profile.api.model;
+
+/**
+ * Created by alex on 26.02.17.
+ */
+public class Contact {
```

Comment:
@Alexey007gold lets rename this package to DTO. And add DTO suffix to all classes in this package. It will be better naming strategy.